### PR TITLE
adding attr_accessible

### PR DESCRIPTION
**Pull request by unorthodoxgeek**

### CODE REVIEW

**Comment by unorthodoxgeek** [comment]
is there an issue number for attr_accessible on docker?

**Comment by daviddavis** [comment]
@unorthodoxgeek there is now http://projects.theforeman.org/issues/12726

**Comment by daviddavis** [comment]
Build: http://ci.theforeman.org/job/test_plugin_pull_request/1846/

File: app/models/docker_container_wizard_states/environment.rb [hunk 1]
```diff
@@ -2,6 +2,8 @@ module DockerContainerWizardStates
   class Environment < ActiveRecord::Base
     self.table_name_prefix = 'docker_container_wizard_states_'
     belongs_to :wizard_state, :class_name => DockerContainerWizardState
+
+    attr_accessible :tty, :docker_container_wizard_state_id, :attach_stdin, :attach_stdout, :attach_stderr
```
Comment (daviddavis) [inline app/models/docker_container_wizard_states/environment.rb]: Looks like this line is too long:

http://ci.theforeman.org/job/test_plugin_pull_request/1846/database=sqlite3,label=fast,ruby=2.1/console

**Comment by daviddavis** [comment]
Build: http://ci.theforeman.org/job/test_plugin_pull_request/1847/

**Comment by daviddavis** [comment]
Merged as 79287e7. Thanks @unorthodoxgeek!

**Pull request closed without merging**